spec/integrations/rails3_request_spec.rb

```diff
@@ -3,6 +3,7 @@
 describe Bugsnag::Middleware::Rails3Request do
   before(:each) do
     Bugsnag.configuration.middleware.use(described_class)
+    stub_const('::Rails::VERSION::STRING', 'test_version')
```

Comment:
Why do we use this for Rails and define a const for Que?

Reply:
This should be used where possible.  I've updated the Que version, and although that still requires a sample module for mocking purposes, it simplifies the tests.